sdk/core/src/services/BloomVaultStorageService.ts

```diff
@@ -233,15 +242,32 @@ export default class BloomVaultStorageService {
   }
 
   public async searchCredentials(storageRegion: string, types?: string[][]): Promise<any[]> {
-    const accessToken = await this._authorizeVcVault(storageRegion)
+    let migration: any = { status: undefined }
+    try {
+      migration = await this._migrationHelper.getMigrationStatus()
+    } catch (err) {
+      console.log('@@@Vault-migration-service migration status call ends with error: ', err)
+    }
 
-    const credentials = await this._fetchAllDecryptedCredentials(accessToken, storageRegion)
+    if (migration.status === 'done') {
+      return []
+    } else {
+      const accessToken = await this._authorizeVcVault(storageRegion)
 
-    if (!types) {
-      return credentials
-    }
+      const credentials = await this._fetchAllDecryptedCredentials(accessToken, storageRegion)
```

Comment:
probably we may need extend `_fetchAllDecryptedCredentials` to return `id` from bloom vault along with credential for us to avoid putting same credentials into queue in case of retry

Reply:
fixed